src/Playground.jl

```diff
@@ -18,92 +20,71 @@ include("clean.jl")
 
 
 export
-    # methods
-    main,
-    argparse,
-    load_config,
-    install,
-    dirinstall,
-    #gitinstall,
-    create,
-    activate,
-    execute,
-    list,
-    clean,
-    config_path,
+    # # methods
+    # main,
+    # load_config,
+    # install,
+    # dirinstall,
+    # gitinstall,
+    # create,
+    # activate,
+    # execute,
+    # list,
+    # clean,
+    # config_path,
 
     # Constants
     DEFAULT_CONFIG
 
 
-
-function main(cmd_args=ARGS, config="", root="")
-    if config == "" && root == ""
-        config = joinpath(config_path(), "config.yml")
-        root = config_path()
-    end
-
+function main{S<:AbstractString}(cmd_args::Array{S}=ARGS, root::AbstractString="")
```

Comment:
How does using ARGS as a parameter default break compiled binaries? How is this different from the current code: `cmd_args=ARGS`?